include/cantera/thermo/IdealSolidSolnPhase.h

```diff
@@ -89,7 +89,7 @@ class IdealSolidSolnPhase : public ThermoPhase
      * partial molar volume solution mixture with pure species phases which
      * exhibit zero volume expansivity:
      * @f[
-     * \hat s(T, P, X_k) = \sum_k X_k \hat s^0_k(T)  - \hat R \sum_k X_k log(X_k)
+     * \hat s(T, P, X_k) = \sum_k X_k \hat s^0_k(T)  - \hat R \sum_k X_k \log(X_k)
```

Comment:
I think almost all of these instances of `\log` should actually be `\ln`. There are a few exceptions, though.

Reply:
Fair point. I believe all instances touched by this PR use `\ln`. As an aside, there are quite a few `\log` instances that are untouched by this PR, which I am leaving for Cantera/enhancements#6.